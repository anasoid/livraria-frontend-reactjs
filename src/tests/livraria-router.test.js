import React from 'react';
import renderer from 'react-test-renderer';
// import LivrariaRouter from '../livraria-router';
// import {MemoryRouter} from 'react-router';

describe('test LivrariaRouter component', () => {

  it('should renders without crashing ', () => {
    // const tree = renderer.create(
    //   <MemoryRouter>
    //     <LivrariaRouter/>
    //   </MemoryRouter>
    // ).toJSON();
    // expect(tree).toMatchSnapshot();
  });

});