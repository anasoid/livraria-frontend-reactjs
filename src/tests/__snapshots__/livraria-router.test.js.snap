// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test LivrariaRouter component should renders without crashing  1`] = `
<div>
  <header>
    <nav
      className="menu"
    >
      <ul>
        <li>
          <a
            href="/livros"
            id="livroItem"
            onClick={[Function]}
          >
            Livros
          </a>
        </li>
        <li>
          <a
            href="/autores"
            id="autorItem"
            onClick={[Function]}
          >
            Autores
          </a>
        </li>
        <li>
          <a
            href="/vendas"
            id="vendaItem"
            onClick={[Function]}
          >
            Vendas
          </a>
        </li>
        <li>
          <a
            href="/login"
            id="logoutItem"
            onClick={[Function]}
          >
            Logout
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <div>
    <div>
      <span
        id="messages"
      />
      <form>
        <fieldset>
          <legend>
            Login
          </legend>
          <label
            htmlFor="loginEmail"
          >
            Email:
            <span>
              *
            </span>
          </label>
          <input
            id="loginEmail"
            name="email"
            type="email"
          />
          <br />
          <label
            htmlFor="loginSenha"
          >
            Senha:
            <span>
              *
            </span>
          </label>
          <input
            id="loginSenha"
            name="senha"
            type="password"
          />
          <br />
          <button
            type="submit"
          >
            Efetue Login
          </button>
        </fieldset>
      </form>
    </div>
  </div>
</div>
`;
